Tests/LibWeb/AccessibilityBridgeMac/tests/test_states.py

```
@@ -0,0 +1,151 @@
+        self.assertIn("AXPress", actions, f"expected AXPress in link actions, got {actions}")
+
+    def test_checkbox_advertises_press_action(self):
+        """Checkboxes expose AXPress (toggling them is a press, not a value-set)."""
+        cb = wait_for_descendant_by_role(self.web, "AXCheckBox", title="Checkbox")
+        self.assertIsNotNone(cb)
+        actions = _action_names(cb)
+        self.assertIn("AXPress", actions, f"expected AXPress in checkbox actions, got {actions}")
+
+    def test_disabled_button_still_advertises_press_action(self):
+        """A disabled button's action list is unchanged. Apple's design pattern is that AXEnabled tells AT clients the
+        action is currently unavailable; the action list itself stays stable – so AT can still describe what *would*
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)